docs/Configuration.md

```diff
@@ -147,3 +147,70 @@ they get annotated with the reason and comment from the resolution.
 
 For details about the available reasons, see
 [ErrorResolutionReason.kt](../model/src/main/kotlin/config/ErrorResolutionReason.kt).
+
```

Comment:
Sorry, I saw to late that the docs were just moved. I'll keep my comments for reference.

Reply:
Applied some of them anyway.